### PR TITLE
add changes

### PR DESCRIPTION
**Pull request by MohamedGood**
fist of five 1

Very hard quiz

### CODE REVIEW

**Comment by MohamedGood** [comment]
A little better understanding

File: README.md [hunk 1]
```diff
@@ -26,7 +26,12 @@ Demonstrate calling the method with an argument of "young prince".
...
 Write your code here:
 ```ruby
-# code here
+def offerRose(person)
+
+  puts "Would you take this rose and help out
+  an old beggar, #{person}?"
+   end
+   offerRose("young prince")
 ```
```
Comment (ebirving) [inline README.md:35]: :+1:

File: README.md [hunk 2]
```diff
@@ -40,16 +45,21 @@ town = {
...
 
 Write your code here:
-```ruby
-# code here
+```
+  town[:residents].delete("Belle")
+
+  town[:castle][:guests].push("Belle")
+
 ```
```
Comment (ebirving) [inline README.md:63]: :+1:

File: README.md [hunk 3]
```diff
@@ -40,16 +45,21 @@ town = {
     num_rooms: 47,
     residents: "Robby Benson",
     guests: []
-  }
-}
+  {
+    }
```
Comment (ebirving) [inline README.md:49]: Be very careful about accidentally changing starter code like this!

File: README.md [hunk 4]
```diff
@@ -71,7 +81,10 @@ Belle is friends with Mrs. Potts
 
 Write your code here:
 ```ruby
-# code here
+friends = ["Chip Potts", "Cogsworth", "Lumière", "Mrs. Potts"]
+friends.each do |person|
+ puts "Belle is friends with #{person}"
+end
 ```
```
Comment (ebirving) [inline README.md:88]: :+1:

File: README.md [hunk 5]
```diff
@@ -89,7 +102,9 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+ERD is short for entity-relationship diagram. ERD is a data modeling technique that visually depicts an information system's enitities and the relationship between them.
+An example of ERD would be a Genie lives in a lamp. A Genie is trapped in the lamp unless a person releases the genie.
+Another example would be a person owns a pet. A pet is the property and possession of a person. A person can have many pets. But a pet can only have one owner.
 ```
```
Comment (ebirving) [inline README.md:108]: :+1:

File: README.md [hunk 6]
```diff
@@ -100,7 +115,9 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is the representation of a plan or theory in the form of a outline or model.
+We represent 1-many relationships in a example like Orange Tree vs. oranges. An orange tree can grow many oranges. But oranges can only grow on a orange tree.  
+
 ```
```
Comment (ebirving) [inline README.md:121]: Great example! In the context of web development, a schema has a more specific definition. It maps your ERD onto the database by describing the entities' attributes and relationships as tables, in the language of the database. In our case, that language is SQL.

**Comment by ebirving** [comment]
The solution branch for the last questions is still being finished -- keep an eye on Slack. Please dedicate as much time as possible to reviewing Ruby basics (Code Academy, etc.)! I also _highly_ recommend that you read through the class notes and repeat in-class examples to build up your understanding of concepts through repetitive practice.

**Pull request closed without merging**